### ISSUE TITLE
Issue with `os_icon` due to a recent commit

### ISSUE THREAD

**Issue by ulf7z** [romkatv/powerlevel10k#23]
Hi,

A recent commit seems to cause issues with `os_icon` on my setup. 
It now prints Tux icon instead of my distribution icon (Arch Linux). It was working fine two days ago, I haven't made any change.

I'm using the [AUR package](https://aur.archlinux.org/packages/zsh-theme-powerlevel10k-git/) and update powerlevel10k with [yay](https://github.com/jguer/yay/) and `"devel": true"` in my .config file.

- #### .zshrc

```
POWERLEVEL9K_MODE='nerdfont-complete'
POWERLEVEL9K_LEFT_PROMPT_ELEMENTS=(os_icon root_indicator dir dir_writable rbenv vcs)
POWERLEVEL9K_RIGHT_PROMPT_ELEMENTS=(status time)

source /usr/share/zsh-theme-powerlevel10k/powerlevel10k.zsh-theme
source /usr/share/zsh-theme-powerlevel10k/gitstatus/gitstatus.plugin.zsh
```

Using powerlevel9k (`source /usr/share/zsh-theme-powerlevel9k/powerlevel9k.zsh-theme`) solves the problem, the Arch icon is back.

##### `$ print $OS_ICON`

- with powerlevel10k:

```
        # Tux icon
```
![Tux_icon](https://user-images.githubusercontent.com/48563433/54374972-168efc00-4678-11e9-8f1e-ec986a505997.png)

- with powerlevel9k: 

```
         # Arch Linux icon
```
![Arch_icon](https://user-images.githubusercontent.com/48563433/54374995-1f7fcd80-4678-11e9-86ea-db2956475a36.png)

I don't have any issue with icons, here's a screenshot with `icons_test` set in `.zshrc`. As you can see, `LINUX_ARCH_ICON` is correctly displayed (bottom left corner, pink background).

![Screenshot_20190314_165505](https://user-images.githubusercontent.com/48563433/54375217-8b623600-4678-11e9-90d4-ba69bc16ebae.png)

- ##### `$ zsh --version`     

```
zsh 5.7.1 (x86_64-pc-linux-gnu)
```
- ##### `$ yay -Qi zsh-theme-powerlevel10k-git `

```
Name            : zsh-theme-powerlevel10k-git
Version         : v0.6.6.r217.gada73dc-1
```

- ##### `$ yay -Qi nerd-fonts-complete`

```
Name            : nerd-fonts-complete
Version         : 2.0.0-5
```

- ##### `$ uname -a`

```
Linux archlinux 5.0.1-arch1-1-ARCH #1 SMP PREEMPT Sun Mar 10 10:55:53 UTC 2019 x86_64 GNU/Linux
```

- ##### `$ cat /usr/lib/os-release`

```NAME="Arch Linux"
PRETTY_NAME="Arch Linux"
ID=arch
BUILD_ID=rolling
ANSI_COLOR="0;36"
HOME_URL="https://www.archlinux.org/"
DOCUMENTATION_URL="https://wiki.archlinux.org/"
SUPPORT_URL="https://bbs.archlinux.org/"
BUG_REPORT_URL="https://bugs.archlinux.org/"
```

Thanks in advance for your help.

**Comment by romkatv** [comment]
Damn, that was an embarrassing typo. Fixed now. I can imagine how horrible you felt when you had to stare at the generic Linux tux instead of the fabulous arch logo :-D

Kudos for the super high quality bug report.

**Comment by ulf7z** [comment]
Fixed indeed by [`af9c7ad`](https://github.com/romkatv/powerlevel10k/commit/af9c7adfd0b3df862a571f2a9257cc733ba03436).

Thank you! keep up the good work.
